client/my-sites/checkout/checkout-thank-you/index.jsx

```diff
@@ -411,8 +414,13 @@ export class CheckoutThankYou extends React.Component {
 			/* eslint-enable wpcalypso/jsx-classname-namespace */
 		}
 
-		// Rebrand Cities thanks page
+		if ( wasDIFMProduct ) {
```

Comment:
Can we add a test for this in https://github.com/Automattic/wp-calypso/blob/trunk/client/my-sites/checkout/checkout-thank-you/test/index.js ?

Reply:
Added two tests to evaluate this condition. :)